db/schema.rb

```diff
@@ -434,7 +437,10 @@
     t.integer "display_order", null: false
     t.boolean "is_internal", null: false
     t.boolean "admin_editable", default: true, null: false
-    t.index ["facility_id", "name"], name: "index_price_groups_on_facility_id_and_name", unique: true
+    t.datetime "created_at"
```

Comment:
I think these might have snuck in from another PR. Not a big deal and we can clean this up when we're merging.